packages/api/src/schema/schema.graphql

```diff
@@ -181,6 +189,8 @@ input HookInput {
   bitbucketUsername: String
   bitbucketToken: String
   bitbucketBuildName: String
+  slackEventFilter: String
+  slackBranchFilter: [String]
```

Comment:
That allows `[null]`, which is not something we want I guess :) 

How about `[String!]`?

Reply:
I see that the whole Hook type doesn't contain required fields. I thought this some tricky plan, so I leave new fields not required too. 

```
type Hook {
  hookId: String
  url: String
  headers: String
  hookEvents: [String]
  hookType: String
  githubToken: String
  githubContext: String
  bitbucketUsername: String
  bitbucketToken: String
  bitbucketBuildName: String
  slackEventFilter: String
  slackBranchFilter: [String]
}
```

Should I add a required flag only to my fields or for all relevant fields? BTW branch filter can be empty or null, but slackEventFilter must have a value.

Reply:
Also empty or null slackBranchFilter means all branches are tracked. So it's a valid state.

Reply:
I have left it as is.